azure/iac/web.rego

```diff
@@ -54,7 +54,7 @@ https_only = false {
 
 https_only_err = "Microsoft.web/Sites resource property httpsOnly missing in the resource" {
     azure_attribute_absence["https_only"]
-} else = "Azure App Service Web app does not redirect HTTP to HTTPS" {
+} else = "Azure Web Service Web app does not redirect HTTP to HTTPS" {
```

Comment:
@shahinshirvani existing message was correct i believe. web app is an Azure App service.